model/task_lifecycle.go

```diff
@@ -733,6 +685,36 @@ func UpdateBuildAndVersionStatusForTask(taskId string, updates *StatusChanges) e
 		"num_tasks":     len(buildTasks),
 	})
 
+	startPhaseAt = time.Now()
+	for _, displayTask := range cache.List() {
+		if err = UpdateDisplayTask(displayTask); err != nil {
+			return errors.Wrap(errors.WithStack(err), "error updating display task")
+		}
+		if err = build.UpdateCachedTask(displayTask, 0); err != nil {
```

Comment:
would it make sense to add an updated display task to `buildTasks` and update all the cached tasks together when we call `UpdateCachedTasks(buildTasks)`?

Reply:
Good catch, updated